setup.cfg

```diff
@@ -22,7 +22,7 @@ install_requires =
     botocore
     click>=8.0.0
     dask
-    datacube
+    datacube<=1.8.13
```

Comment:
As this issue has been fixed in develop and will therefore definitely be fixed in 1.8.15, maybe just require `!=1.8.14`